apps/documentation/src/app/pages/primitives/file-upload.md

```diff
@@ -21,7 +21,11 @@ import { NgpFileUpload } from 'ng-primitives/file-upload';
 Assemble the file-upload directives in your template.
 
 ```html
-<button ngpFileUpload (ngpFileUploadSelected)="onFilesSelected($event)"></button>
+<button
+  ngpFileUpload
+  (ngpFileUploadSelected)="onFilesSelected($event)"
+  (ngpFileUploadCanceled)="onCancel()"
```

Comment:
Could we add this output to the list of inputs/outputs below?

Reply:
Sorry I missed that, have pushed an update